beacon-chain/db/kv/slasher.go

```diff
@@ -173,3 +173,29 @@ func (s *Store) SaveSlasherChunks(
 		return nil
 	})
 }
+
+// Encode an indexed attestation's required fields for slashing protection into bytes.
+func encodeIndexedAttestationRecord(att *ethpb.IndexedAttestation, signingRoot [32]byte) ([]byte, error) {
+	if att == nil || att.Data == nil || att.Data.Target == nil || att.Data.Source == nil {
```

Comment:
A good first issue is to add `ValidateNilIndexedAttestation` helper. Something to add when we are bored